test/test-applications/integrations/dependency-libs/Samples.SqlServer.Vb/Samples.SqlServer.Vb.vbproj

```diff
@@ -1,9 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0;netcoreapp3.1</TargetFrameworks>
-
+    <TargetFrameworks>net45;net461;netstandard2.0;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Isn't this the "default" set of frameworks?

Reply:
This is using `net45` instead of `net452`. But I think it can use `net452` just like the C# project, so we could use the default frameworks here

Reply:
Done in c3aee2b4242d603e045ce26dcdb943f9b5efd47b.

Reply:
These have an empty props so they need to specify target. Fixed in 94de9af8.